content/blog/2018-11-11-colocr.md

```diff
@@ -0,0 +1,106 @@
+---
+slug: "colocr"
+title: Co-localization analysis of image microscopy
+package_version: 0.1.0
+authors:
+  - name: Mahmoud Ahmed
+    url: https://github.com/MahShaaban
```

Comment:
```suggestion
    url: https://github.com/MahShaaban
    twitter: Ma7moudsha3ban
```

Reply:
If you want, making this change to YAML would provide a link to your twitter profile via twitter icon as in this example: https://ropensci.org/blog/2018/11/13/antarctic/